apps/emqx_auth_http/src/emqx_http_client.erl

```diff
@@ -191,7 +191,7 @@ handle_info(Info, State) ->
     {noreply, State}.
 
 terminate(_Reason, #state{pool = Pool, id = Id}) ->
-    gproc_pool:disconnect_worker(Pool, {Pool, Id}),
+    gropc:disconnect_worker(Pool, {Pool, Id}),
```

Comment:
```suggestion
    gproc_pool:disconnect_worker(Pool, {Pool, Id}),
```

Reply:
And, I updated the code of auth-http dev/v4.3.0. to prevent this issue from occurring the next time

Reply:
will sync again and update this PR